hassio-google-drive-backup/backup/static/layouts/partials/navbar.jinja2

```diff
@@ -0,0 +1,133 @@
+{% if coordEnabled %}
+{% include 'layouts/partials/modals/snapshot/delete.jinja2' %}
+{% include 'layouts/partials/modals/snapshot/new.jinja2' %}
+{% include 'layouts/partials/modals/snapshot/retain.jinja2' %}
+{% include 'layouts/partials/modals/snapshot/upload.jinja2' %}
+{% include 'layouts/partials/modals/snapshot/restore.jinja2' %}
+{% include 'layouts/partials/modals/snapshot/details.jinja2' %}
+{% endif %}
+
+{% include 'layouts/partials/modals/settings.jinja2' %}
+{% include 'layouts/partials/modals/help.jinja2' %}
+{% include 'layouts/partials/modals/bug.jinja2' %}
+
+{% if coordEnabled %}
+<script type="text/javascript">
+  $(document).ready(function () {
+    $('#deletemodal').modal();
+    $('#snapshotmodal').modal();
+    $('#retainmodal').modal();
+    $('#uploadmodal').modal();
+    $('#restoremodal').modal();
+    $('#details_modal').modal();
+  });
+</script>
+{% endif %}
+
+<script type="text/javascript">
+  $(document).ready(function () {
+    $('#settings_modal').modal({
+      dismissible: false
+    });
+    $('#help_modal').modal();
+    $('#bug_modal').modal();
+    $('.tooltipped').tooltip();
+    $('.collapsible').collapsible();
+    $('.sidenav').sidenav();
+
+    const dropdownTriggers = document.querySelectorAll('.dropdown-trigger');
+    const instances = M.Dropdown.init(dropdownTriggers, { 'constrainWidth': false });
+  });
+</script>
+
+
+{% macro actionMenu(id) %}
+<ul id="{{ id }}" class="dropdown-content">
+  <li>
+    <a onclick="newSnapshotClick(); return false;" style="cursor: pointer">
+      <i class="material-icons left">add</i>Snapshot Now
+    </a>
+  </li>
+  <li>
+    <a onclick="sync('error_card')" style="cursor: pointer"><i class="material-icons left">autorenew</i>Sync Now</a>
+  </li>
+  <li><a class="open_drive_link" target="_blank"><i class="material-icons">cloud</i>Open Drive Snapshots</a></li>
```

Comment:
This link only works on pages that periodically call /getstatus.  If the navbar is going to be displayed on the privacy policy and terms of service pages, either this item should be removed.

Reply:
When you say
> on pages that periodically call /getstatus
you mean only in the `working.jinja2` page? Or is there any other page?

Reply:
working.jinja2 and index.jinja2 are the only pages that call /getstatus so now that I think of it the logs page also shouldn't have the link to Google Drive

Reply:
ok I can add a condition for that